docs/pipeline.md

```diff
@@ -58,10 +58,10 @@ Indic scripts only:
 
 ---
 PR preparation:
-- [ ] Proposal document — Cite L2 number
-- [ ] UTC decisions — Cite _in extenso_
+- [ ] If a Proposal document — Cite L2 number in the format XXXX
+- [ ] If for a UTC decision — Cite in the format UTC-\d\d\d-[MC]\d+
```

Comment:
I think we want both of these, not either-or --> so I wouldn't add the "If" -- I would leave the parts before the em dashes as is.

Reply:
Seconded.